.gitignore

```diff
@@ -116,4 +116,5 @@ UnityProject/ProjectSettings/Packages/com.xeleh.enhancer/Settings.asset
 UnityProject/AddressablePackingProjects/SoundAndMusic/.vscode/settings.json
 UnityProject/Assets/Resources/ScriptableObjectsSingletons/SpriteCatalogueSingleton.asset
 Tools/CodeScanning/CodeScan/CodeScan/obj
-UnityProject/AddressablePackingProjects/SoundAndMusic/obj
\ No newline at end of file
+UnityProject/AddressablePackingProjects/SoundAndMusic/obj
```

Comment:
why are you editing this?

Reply:
just revert it then I'll approve

Reply:
I, uh… forgor that reverts existed.
Will fix by today, unless git CLI explodes me noggin.

Reply:
Changes to .gitignore should be reverted now.